doc/usage/advanced/intl.rst

```diff
@@ -119,6 +119,11 @@ section describe an easy way to translate with *sphinx-intl*.
       $ make gettext
 
    The generated pot files will be placed in the ``_build/gettext`` directory.
+   If you want to customize the output beyond what can be done via the
+   :ref:`intl-options`, the
+   :download:`default pot file template <../../../sphinx/templates/gettext/message.pot.jinja>`
+   can be replaced by a custom :file:`message.pot.jinja` file placed in any
+   directory listed in :confval:`templates_path`.
```

Comment:
```suggestion
+
+   .. tip:: Customizing the ``.pot`` output
+
+      The POT files are generated using a
+      :download:`built-in Jinja template </../sphinx/templates/gettext/message.pot.jinja>`.
+      You can customize their format by copying that file to a directory in
+      :confval:`templates_path` and modifying it accordingly.
```

Reply:
It's a little unusual that we're including a file from outside of the `doc` directory here.  I _think_ it's OK, but wanted to mention it in case other reviewers want to inspect that too.  My suggested edit here uses `/../` to refer to it as one-directory-above the source root (ref the [`:download:` directive docs](https://www.sphinx-doc.org/en/master/usage/referencing.html#role-download)).

Reply:
Yeah, I wasn't sure at first if that would run afoul of any security restrictions, but it worked fine in the output (into which the file is just copied).  I'm fine with the proposed change as long as that also renders correctly (it just looked even stranger to me to have a path that says essentially "one level above root").